src/main/kotlin/com/dcd/server/core/domain/domain/service/impl/RemoveHttpConfigServiceImpl.kt

```
@@ -1,27 +1,31 @@
     override fun removeHttpConfig(domain: Domain) {
         if (domain.application == null)
             throw DomainNotConnectedException()
 
-        val httpConfigDirectory = "${domainConfigPath}/nginx/conf/${domain.id}"
-        val commandResult = commandPort.executeShellCommand("rm -r '$httpConfigDirectory'")
+        val httpConfigDirectory = Paths.get(domainConfigPath, "nginx", "conf", domain.id)
 
-        if (commandResult.exitValue != 0)
+        try {
+            fileOperationPort.deleteDirectory(httpConfigDirectory)
+        } catch (e: FileOperationException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)